File: Beecrowd/1116 Dividindo X por Y/Main.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) throws Exception {
        Scanner sc = new Scanner(System.in);

        int N, X, Y;

        N = sc.nextInt();

        for(int i = 0; i < N; i++){
            X = sc.nextInt();
            Y = sc.nextInt();

            if(Y == 0)
                System.out.println("divisao impossivel");
            else
                System.out.printf("%.1f\n", (double) X / Y);
        }

        sc.close();
    }
}

File: Beecrowd/1116 Dividindo X por Y/main.py
N = int(input())

for i in range(N):
  entrada = [int(a) for a in input().split()]
  X = entrada[0]
  Y = entrada[1]
  if Y == 0:
    print("divisao impossivel")
  else:
    D = X / Y
    print("%0.1f" %D)